src/coffee/run.coffee

```diff
@@ -103,8 +119,8 @@ Promise.all [readCsvAsync(argv.types), readCsvAsync(argv.attributes)]
         process.exit 0
   .catch (e) ->
     console.error "Oops, something went wrong: #{e.message}"
-    process.exit 1
```

Comment:
hard to see where exactly the errors occur, was just for debug, will fix it. Just curious, is there a good way to see where an error happens with this setup?